submission/submission-config/src/main/kotlin/ac/uk/ebi/biostd/common/properties/SecurityProperties.kt

```diff
@@ -26,14 +26,16 @@ data class FilesProperties(
     val defaultMode: StorageMode,
     val filesDirPath: String,
     val magicDirPath: String,
+    var userFtpDirPath: String,
```

Comment:
it would be clearer to use `ftpInDirPath` for user files upload related code and `ftpOut` for the location where the submission files will be stored